wire-ios/Wire-iOS/Sources/UserInterface/Components/NetworkStatus/NetworkStatusView.swift

```diff
@@ -211,9 +198,10 @@ final class NetworkStatusView: UIView {
 
     func updateUI(animated: Bool) {
         log(networkStatus: state)
-        // When the app is in background, hide the sync bar and offline bar. It prevents the sync bar is "disappear in a blink" visual artifact.
         var networkStatusViewState = state
-        if application.applicationState == .background {
+
+        // When the app is in background, hide the sync bar and offline bar. It prevents the sync bar is "disappear in a blink" visual artifact.
+        if let activationState = window?.windowScene?.activationState, ![.foregroundActive, .foregroundInactive].contains(activationState) {
```

Comment:
The intention here is to hid the bar for both `.background` and `.unattached` states right?